### PR TITLE
Custom perameter inputs

### LINKED ISSUE #7: As a user I need a way to be able to select the maximum weather restrictions for my personal aircraft so that I am able to quickly visually see if my specific aircraft has optimal flying conditions.

### PR DESCRIPTION
**Pull request by JesseMorganMet**
This pull request has a feature that allows a user to change the parameters for the optimal weather conditions checker, i.e. you can change the wind speed from default values to whatever your specific model aircraft can handle.

### CODE REVIEW

File: src/app/app.component.spec.ts [hunk 1]
```diff
@@ -49,5 +52,27 @@ describe('AppComponent', () => {
...
+    // describe('btnClick', () => {
+    //   it('Should Add objects to an object', async () => {
+    //     component.weatherInputs = {wind: 5};
+    //     component.precipitationInput = 50;
+    //     component.visibilityInput = 15;
+    //
+    //     component.btnClick();
+    //     expect(component.weatherInputs).toBe({wind: 5, rain: 50, vis: 15});
+    //   })
+    // })
```
Comment (mo-chrisbennett) [inline src/app/app.component.spec.ts]: Are those the tests that you are stuck on?

Comment (JesseMorganMet) [reply src/app/app.component.spec.ts]: Yes, well setDefaultForm im stuck on. 
btnClick currently does nothing, I forgot to remove the test contents

File: src/app/app.component.spec.ts [hunk 2]
```diff
@@ -14,6 +15,12 @@ describe('AppComponent', () => {
   let httpTestingController: HttpTestingController;
   let mockData = {features: [{properties: "any"}]};
   let mockResult: any = [];
+  // let hello = {wind:'',rain:'',vis:''}
```
Comment (mo-chrisbennett) [inline src/app/app.component.spec.ts]: If you're not using that anymore, you might as well remove it?

File: src/app/app.component.spec.ts [hunk 3]
```diff
@@ -49,5 +56,11 @@ describe('AppComponent', () => {
         })
       })
     })
+
+    //outdated test placeholder til button updated
+    // describe('btnClick', () => {
+    //   it('Should ', async () => {
+    //   })
+    // })
```
Comment (mo-chrisbennett) [inline src/app/app.component.spec.ts]: Will the button be updated?

**Review by mo-chrisbennett — APPROVED** [review]
Other than commented out code, this looks ok to me.

File: src/app/app.component.spec.ts (hunk 3 above)
Comment (JesseMorganMet) [reply src/app/app.component.spec.ts]: to be honest, i could probably remove the button, I will have a button with different functionality at some point. but currently in this feature the button will not have any functionality

**Pull request merged**